src/telemetry/telemetry.controller.spec.ts

```diff
@@ -66,6 +66,7 @@ describe('TelemetryController', () => {
     usersService = app.get(UsersService);
     versionsService = app.get(VersionsService);
     await app.init();
```

Comment:
These test are failing as a result of this conditional check:
https://github.com/iron-fish/ironfish-api/pull/1256/files#diff-b9ee66213b9bfe61d1f85d4cba47efe234bf537c6c9b05f1b072dfbb1d9a4654R113
Mocking test time to make tests pass

Reply:
This was fixed in another PR. I rebased.